roadmap.md

```diff
@@ -12,7 +12,6 @@ The Microsoft.Data.SqlClient roadmap communicates project priorities for evolvin
 | Milestone                 | Release Date | Project Board |
 |---------------------------|--------------|---------------|
 | Microsoft.Data.SqlClient v1.1 (servicing) | As needed (see also [1.1 releases](https://github.com/dotnet/sqlclient/blob/master/release-notes/1.1)) | Closed |
-| Microsoft.Data.SqlClient v2.0 (servicing) | As needed (see also [2.0 releases](https://github.com/dotnet/sqlclient/blob/master/release-notes/2.0)) | Closed |
```

Comment:
We have 2.0 in release-notes. Should we also keep 2.0 here?

Reply:
2.0 is out of support - means it won't be serviced anymore.
Release notes are not going anywhere.

Reply:
Fair enough.